lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
> @detiber I must be missing something. In the first example, wouldn't 'name' end up being the value 'CentOS-7-x86_64-GenericCloud.qcow2' ?
> 
> And isn't that exactly the behavior we'd be looking for?

Yes, not sure what I was thinking there, I must have mixed up my example and I can't think of what I was trying to convey now.

> Then default filter takes that value and determines if it should change it. In this case, since we passed in 'true' as the second parameter, empty strings do get replaced.
> 
> So, again, wouldn't 'name' end up being the value 'CentOS-7-x86_64-GenericCloud.qcow2' ?
> 
> I agree that in the second example, 'name' would be set to an empty string, but again, that's what we'd expect. I think I'm totally missing your point.

In the second example, the default behavior isn't carried through the possible defaults, where we want to treat the value as a python boolean, so the flag setting the value to "" should be treated as a python False and we should instead use the env set default.